modules/react/tabs/stories/Tabs.stories.mdx

```diff
@@ -22,80 +33,67 @@ yarn add @workday/canvas-kit-react
 
 ## Usage
 
-This component follows the
-[W3 Tabs specification](https://www.w3.org/TR/wai-aria-practices/#tabpanel). `Tabs` is a
-[Compound Component](/getting-started/for-developers/resources/compound-components/), which means it
-can be composed in a variety of different ways. We'll first cover some basic usage examples followed
-by more advanced usage.
+### Basic Example
+
+`Tabs` includes a container `Tabs` component and the following subcomponents: `Tabs.List`,
+`Tabs.Item` and `Tabs.Panel`.
 
-### Basic Usage
+<ExampleCodeBlock code={Basic} />
 
-Below are examples we expect will work for most cases. Please refer to the
-[Advanced Usage](#advanced-usage) section for details on supporting custom implementations.
+### Hoisted Model
```

Comment:
I moved the `Hoisted Model` example right below the `Basic Example`. Previously, it was much further down in the Advanced Usage section (now gone) which was characterized with "we expect most people won't need to read this section," but it seems like a fairly common use case to me. Thoughts?

Reply:
Yeah, I agree. That copy was from a while back. Seems right to move it up. 👍🏻